source/main.c

```diff
@@ -822,10 +823,10 @@ static void get_settings(int disc_type) {
 		while (!(get_buttons_pressed() & (PAD_BUTTON_RIGHT | PAD_BUTTON_LEFT | PAD_BUTTON_A | PAD_BUTTON_UP | PAD_BUTTON_DOWN)));
 		u32 btns = get_buttons_pressed();
 		if(btns & PAD_BUTTON_RIGHT) {
-			toggleOption(currentSettingPos+((disc_type == IS_WII_DISC)?MAX_NGC_OPTIONS:0), 1);
+			toggleOption(currentSettingPos+((disc_type == IS_WII_DISC)?MAX_NGC_OPTIONS + 1:0), 1);
 		}
 		if(btns & PAD_BUTTON_LEFT) {
-			toggleOption(currentSettingPos+((disc_type == IS_WII_DISC)?MAX_NGC_OPTIONS:0), -1);
+			toggleOption(currentSettingPos+((disc_type == IS_WII_DISC)?MAX_NGC_OPTIONS + 1:0), -1);
```

Comment:
I am the least happy with this "+ 1"

I haven't quite wrapped my head around on how this works, but basically since I removed one option of the Wii ripper menu, I need to add +1 to get the position in the array correctly, otherwise the option selection does not work.